docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3'
+
+services:
+  redis:
+    image: redis:3.0.7-alpine
+    container_name: redis
+    command: redis-server --appendonly yes
+
+  compliance-checker:
+    image: test_ccweb:latest
+    container_name: compliance-checker-web
+    env_file: .env
+    volumes:
+      - "./contrib/config/config.yml:/usr/lib/ccweb/config.local.yml"
+    ports:
+      - "3000:3000"
```

Comment:
This compose file looks more suited for deployment than development. We may need new dockerfiles for development and I'd imagine individual services for the frontend, the flask app, and the service worker, running in debug/development mode (ie auto reload on file changes).